openapi.yaml

```diff
@@ -551,6 +619,15 @@ components:
           type: string
           x-stoplight:
             id: 73fbvgg6mhsid
+        feedback:
```

Comment:
Inside pal-api, we have a separate `MessageWithFeedback` type, but we don't return messages without feedback anywhere in the API right now.